src/core/cluster.py

```diff
@@ -395,6 +401,20 @@ def bootstrap_server(self) -> str:
             )
         )
 
+    @property
+    def controller_quorum_uris(self) -> str:
+        """The current controller quorum uris when running KRaft mode."""
+        # FIXME: when running controller node.id will be unit.id + 100. If unit is only running
+        # the broker node.id == unit.id. This way we can keep a human readable mapping of ids.
+        if self.kraft_mode and self.runs_controller:
+            return ",".join(
+                [
+                    f"{broker.unit_id+100}@{broker.host}:{CONTROLLER_PORT}"
```

Comment:
**suggestion**: What are your thoughts on implementing the +100 logic directly in the `unit_id` property?

Reply:
This was removed from this place. Will be reworked when we tackle large deployments. I've placed the offset as a literal for now.